R/slider.R

```diff
@@ -98,12 +98,12 @@ range_input <- function(input_id, value, value2, min, max, step = 1, class = NUL
 #' @param inputId Input name.
 #' @rdname slider
 #' @export
-rangeInput <- function(inputId, label, min, max, value, step = 1, width = NULL, ...) {
+rangeInput <- function(inputId, label, min, max, value, value2, step = 1, width = NULL, ...) {
```

Comment:
I've just realized such function doesn;t exist on shiny. I'll create a separate issue regarding that,

Reply:
yes, we should probably remove this function